README.md

```diff
@@ -83,6 +83,11 @@ This library is intended to provide a quicker and easier way to get started usin
 * `uint8_t getAddress(void)`<br>
   Returns the current I&sup2;C address.
 
+* `bool begin(void)`<br>
+  Iniitializes the sensor to use Wire as the default port. 
+* `bool begin(&Wire1)`<br>
+  Iniitializes the sensor to use Wire1 as the default port or change Wire1 to Wire2, Wire3 etc.
```

Comment:
"Initializes the two wire interface to something other than the default Wire1 (for example, Wire2 or Wire3)."